package-lock.json

```diff
@@ -9,201 +9,201 @@
             "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.10.5.tgz",
             "integrity": "sha512-j9H9qSf3kLdM0Ao3aGPbGZ73mEA9XazuupcS6cDGWuiyAcANoguhP0r2Lx32H5JGw4sSSoHG3x/mxVnHgvOoyA==",
             "requires": {
-                "chokidar": "^2.1.8",
```

Comment:
@theywa, next time no changes in package-lock.json please (other than for modules in package.json that you changed). It makes merging a pain.

Reply:
sorry about that, I will keep that in mind.